src/Listing/index.tsx

```diff
@@ -187,7 +210,27 @@ const Listing = ({
       return '100%';
     }
 
-    return `calc(100vh - ${tableTopOffset}px - ${paginationElement.current?.clientHeight}px - ${tableHeaderElement.current?.clientHeight}px)`;
+    return `calc(100vh - ${tableTopOffset}px - ${
+      actionBarRef.current?.offsetHeight
+    }px - ${headerHeight}px - ${loadingIndicatorHeight}px - ${theme.spacing(
+      1,
+    )}px)`;
+  };
+
+  const getGridTemplateColumn = (): string => {
+    const checkbox = checkable ? 'auto ' : '';
```

Comment:
```suggestion
    const checkbox = checkable ? 'min-content' : '';
```
That avoid the checkbox to take a lot of space

Reply:
don't forget space after `min-content`